### PR TITLE
Update workload page

### PR DESCRIPTION
**Pull request by inyee786**
Get the number of replica and controller from mayaapi-server
Get the openebs version
Get the k8s version
Modify the UI of application status in workload
Make workload right side scrollable

Signed-off-by: inyee786 <intakhab.ali@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
Screenshot from 2019-04-19 18-53-21

### CODE REVIEW

File: src/app/services/litmus.services.ts [hunk 1]
```diff
@@ -6,17 +6,9 @@ import { litmusstatus,litmuslog,jobname } from "../model/data.model";
 @Injectable()
 export class LitmusService {
   private apiurl: string;
-  private host: string;
+
```
Comment (chandankumar4) [inline src/app/services/litmus.services.ts]: This blank line is not required

File: src/app/workloads/workloads.component.html [hunk 2]
```diff
@@ -25,23 +25,22 @@
             <div class="d-flex flex-column">
               <div class="d-flex align-items-center mb-3">
                 <img src="/assets/images/company-logos/k8slogo.svg" alt="" height="30">
-                <p class="mb-0 fo-sz-12 text-dark"> &nbsp;&nbsp;1.10.7</p>
+                <p class="mb-0 fo-sz-12 text-dark"> &nbsp;&nbsp;{{kubeletVersion}}</p>
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.html]: In place of **&nbsp** can we some margin or padding concept

File: src/app/workloads/workloads.component.html [hunk 3]
```diff
@@ -25,23 +25,22 @@
...
-              <!-- <p class=" mb-0 fo-sz-12 workloadImg">Image: {{workloadImage}}</p> -->
             </div>
           </div>
         </div>
         <div class="d-flex flex-column my-auto pr-4">
           <div class="d-flex flex-column ">
             <div class="d-flex align-items-center mb-3 ml-4">
               <img src="/assets/images/company-logos/openebs-logo-mule.svg" alt="" height="30">
-              <p class="mb-0 fo-sz-12 text-dark"> &nbsp;&nbsp; {{openebsengine}} -0.8.1</p>
+              <p class="mb-0 fo-sz-12 text-dark"> &nbsp;&nbsp; {{openebsengine}} - {{openebsversion}}</p>
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.html]: need to replace **&nbsp** with some padding ot margin

File: src/app/workloads/workloads.component.ts [hunk 4]
```diff
@@ -32,20 +32,22 @@ import { ISubscription } from "rxjs/Subscription";
...
+  private podUnsub$: ISubscription;
+  private timeUnsub$: ISubscription;
+  private litmustimeUnsub$: ISubscription;
+  private litmusUnsub$: ISubscription;
   private routeSub: ISubscription;
-  private personalSub: ISubscription;
-  private pvcDetails: ISubscription;
+  private personalSub$: ISubscription;
+  private pvcDetails$: ISubscription;
+  private mayactlVolumeList$: ISubscription;
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.ts]: I think **$** is not required for private variable, standard practice is use only the camel case
ref: https://v2.angular.io/docs/ts/latest/guide/style-guide.html

File: src/app/workloads/workloads.component.ts [hunk 5]
```diff
@@ -277,29 +269,41 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
...
+        this.mayactlVolumeList$ = this.kubernetsServices.getVolumeDetails(this.workloadName,this.openebsEngine,this.mayapvc)
+        .subscribe(res => {
+          let response = Object.entries(res);
+          this.numberController = response.length;
+          for(let i in res){
+            this.numberReplica = this.numberReplica + parseInt(res[i].replicas);
+          }
+        })
       });
+
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.ts]: This blank line is not required

File: src/app/workloads/workloads.component.ts (hunk 4 above)
Comment (inyee786) [reply src/app/workloads/workloads.component.ts]: it's observale

File: src/app/services/litmus.services.ts (hunk 1 above)
Comment (inyee786) [reply src/app/services/litmus.services.ts]: i think is should be good if separate the two thing by line.
the first part contains the variable and below that is a constructor  that's why I put the line here

File: src/app/workloads/workloads.component.ts (hunk 4 above)
Comment (chandankumar4) [reply src/app/workloads/workloads.component.ts]: In observable also they are not using any **$** sign

File: src/app/services/litmus.services.ts (hunk 1 above)
Comment (chandankumar4) [reply src/app/services/litmus.services.ts]: okay!!

File: src/app/workloads/workloads.component.ts [hunk 6]
```diff
@@ -215,13 +217,10 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
           this.pvcarray = this.pvctemp.pvc;
           // this.workloadImage = this.statefullSets[0].dockerImage;
           // this.dockerImage = this.jivaContrllers[0].openebsjivaversion;
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.ts]: These commented line required later?

**Pull request closed without merging**